app/views/smart_proxies/index.html.erb

```diff
@@ -17,7 +22,12 @@
     <% for proxy in @smart_proxies %>
       <tr class="proxy-show" data-url="<%= ping_smart_proxy_path(proxy) %>">
         <td><%= link_to_if_authorized proxy.name, hash_for_smart_proxy_path(:id => proxy).merge(:auth_object => proxy, :authorizer => authorizer) %></td>
-        <td><%= proxy.url %></td>
+        <% if SETTINGS[:locations_enabled] %>
+          <td><%= generate_links_for(proxy.locations).html_safe %></td>
+        <% end %>
+        <% if SETTINGS[:organizations_enabled] %>
+          <td><%= generate_links_for(proxy.organizations).html_safe %></td>
```

Comment:
@dLobatog have you tested this? How it looks like when there are many locations/orgs?

I think we need "ellipsis" here so it does not overflow perhaps?